packages/leavittbook/src/components/titanium-data-table-header.ts

```diff
@@ -0,0 +1,40 @@
+import { LitElement, html, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { h1, h2, h3, h5, p } from '@leavittsoftware/titanium-styles';
+import StoryStyles from '../styles/story-styles';
+import '@leavittsoftware/titanium-data-table/lib/titanium-data-table-header';
+import '@leavittsoftware/titanium-card';
+import '../shared/code-block';
+import '../shared/story-header';
+import '@api-viewer/docs';
+
+@customElement('titanium-data-table-header-demo')
+export class TitaniumDateTableHeaderDemoElement extends LitElement {
+  static styles = [h1, h2, h3, h5, p, StoryStyles, css``];
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Examples using large, right, no-sort, and width attributes</p>
+        <titanium-data-table-header large column-name="Name" large title="Name"></titanium-data-table-header>
```

Comment:
Can we have the Title reflect the property being demo'd? So this one would be "Large",
Also this has the large attribute twice